data/json/items/fake.json

```diff
@@ -58,7 +58,11 @@
     "copy-from": "fake_item",
     "type": "TOOL",
     "name": { "str": "autonomous surgical scalpels", "str_pl": "autonomous surgical scalpels" },
-    "flags": [ "TRADER_AVOID" ],
+    "description": "A system of surgical grade scalpels.  They allow you to make precise cuts and can also be used as a high-quality butchering tool.",
+    "to_hit": -3,
```

Comment:
I'm not sure about this. These scalpels are effectively works as continuation of one's arms, so they could be considered as "fists" in terms of to-hit calculation. If so, according to our `game-balance.md`, they get +2 for their "grip".

Reply:
No, +2 isn't appropriate either. 
All weapons start with -2 to hit. These scalpels get +2 for having a fist-like "grip", 0 for length, -1 for being "line of damage" weapons, and 0 for neutral balance. Total of -1.